rpms/build.sh

```diff
@@ -82,19 +83,16 @@ if [[ "${future}" == "yes" ]]; then
     old_name="wazuh-${build_target}-${base_version}-${package_release}"
     package_name=wazuh-${build_target}-${wazuh_version}
     old_package_name=wazuh-${build_target}-${base_version}
-    cp -r "${specs_path}/${base_version}" "${specs_path}/${wazuh_version}"
-    cd "${specs_path}/${wazuh_version}"
-    rename "${base_version}" "${wazuh_version}" *${base_version}*
-    cd -
+    cp -r "${specs_path}/" "${specs_path}/"
```

Comment:
I dont see this line in the master PR https://github.com/wazuh/wazuh-packages/pull/1571/files

Reply:
Done [c28ed20](https://github.com/wazuh/wazuh-packages/pull/1579/commits/c28ed207f9bcada77bfb5ec27de04e67ea97c9db)